### ISSUE TITLE
[Suggestion] Add a configuration to make anvil behave like vanilla

### ISSUE THREAD

**Issue by LioxAcc** [Shadows-of-Fire/Apotheosis#1346]
I have disabled the Apotheosis's enchantment module, but the anvil still didn't subtract experience levels like in vanilla (it still consumes much less the higher your level is)

Forge 1.20.1 47.3.7
Apotheosis-1.20.1-7.4.2
ApothicAttributes-1.20.1-1.3.7
Placebo-1.20.1-8.6.2
(latest to date)

By the way, I tried to disable ALL modules just in case. The behavior remains the same

**Comment by Shadows-of-Fire** [comment]
I could, though this isn't really high priority. The behavior apoth supplies to the anvil just makes it unnecessary to use the anvil at exactly the consumed xp level instead of needing to minmax by using some kind of XP storage device.

**Comment by LioxAcc** [comment]
I agree that the function makes the process more comfortable and efficient
However, in my personal case, I prefer not to have that function, and keep all the other modules of this mod
Thank you very much for your response
